.github/workflows/ci.yml

```diff
@@ -11,9 +11,12 @@ jobs:
     runs-on: macOS-latest
     strategy:
       matrix:
-        destination: ['platform=iOS Simulator,OS=14.4,name=iPhone 11']
+        destination: ['platform=iOS Simulator,OS=15.2,name=iPhone 11']
```

Comment:
I assume this updates your framework to be built against iOS 15.2 and Xcode version 13.2.1?

Reply:
Yes, but just for the tests